server/src/controllers/user_controller.ts

```diff
@@ -0,0 +1,31 @@
+import "reflect-metadata";
+import { Request, Response } from "restify";
+import { Controller, Get, interfaces } from "inversify-restify-utils";
+import { inject, injectable, named } from "inversify";
+import { authMiddleware } from "./auth_utils";
+import { UserService } from "../services/user_service";
+
+@Controller("/users")
+@injectable()
+export class UserController implements interfaces.Controller {
+    constructor(
+        @inject("Service")
+        @named("UserService")
+        private readonly userService: UserService,
+    ) {}
+
+    @Get("/me", authMiddleware)
```

Comment:
will that be used for Cov-144 The navbar shows a placeholder... for example

Reply:
yessir